models/repo_list.go

```diff
@@ -310,7 +321,13 @@ func userOrgTeamRepoBuilder(userID int64) *builder.Builder {
 func userOrgTeamUnitRepoBuilder(userID int64, unitType unit.Type) *builder.Builder {
 	return userOrgTeamRepoBuilder(userID).
 		Join("INNER", "team_unit", "`team_unit`.team_id = `team_repo`.team_id").
-		Where(builder.Eq{"`team_unit`.`type`": unitType})
+		Where(builder.Eq{"`team_unit`.`type`": unitType}).
+		And(builder.Gt{"`team_unit`.`access_mode`": int(perm.AccessModeNone)})
```

Comment:
I don't think you can create a team_unit with AccessModeNone so I'm not sure that this is needed.

Reply:
Hm, in the user interface I am able to select `No Access`, `Read` and `Write` for each individual Unit, I will perform some additional research to see how these options are stored in table `team_unit`.

Reply:
As far as I can tell, `AccessModeNone` is (sometimes) stored in table `team_unit`. For example, the following settings:
![image](https://user-images.githubusercontent.com/10838836/170967884-fdc2ef4c-497e-4e5f-b145-8dec17eed1a8.png)

lead to the following contents of `team_unit`:
![image](https://user-images.githubusercontent.com/10838836/170968186-b7312fe8-0e5b-4bf3-bd0c-130820a92615.png)

So I believe the check is required here ...

Reply:
OKs